test/epicenter_web/live/profile_live_test.exs

```diff
@@ -878,7 +878,7 @@ defmodule EpicenterWeb.ProfileLiveTest do
 
       contact_investigation_id = contact_investigation.id
       view = Pages.Profile.visit(conn, contact_investigation.exposed_person)
-      initiating_case_text = "Initiated by index case \##{sick_person.id}"
+      initiating_case_text = "Initiated by index case  \##{sick_person.id}"
```

Comment:
Does it need this new space in the UI? that looks to me like it would render as `Initiated by index case  #123` (2 spaces instead of 1).

Reply:
fixed